features/support/defaults.js

```diff
@@ -10,3 +10,13 @@ Before({ tags: '@GenerationService' }, function () {
   request.setBaseUrl(this.parameters.generationServiceUrl);
   settings.setReporterAutoRun(false);
 });
+
+Before({ tags: '@RepositoryService' }, function () {
```

Comment:
I think we might have a race condition with line 4 now - I would get rid of the other Before and the associated config property since we aren't really using the base app URL anymore.

Reply:
The cucmber.js and .env files should also be updated to reflect this. Now you need to set all 3 endpoints (base isn't being used anymore).